hack/pre-release.sh

```diff
@@ -148,6 +159,128 @@ update_OCP_version() {
   fi 
 }
 
+# This function handles updating the versions in GitHub
+# takes 3 arguments
+# 1st arg: base branch to operate on 
+# 2nd arg: stream to update. Can be x, y or z to represent the x y or z stream.#
+# 3rd arg: current WMCO version
+github_update() {
+  if [ "$#" -ne 3 ]; then
+    echo incorrect parameter count for github_update $#
+    return 1
+  fi
+
+  base_branch=$1
+  stream=$2
+  wmco_version=$3
+
+  # check if the git branch is dirty, and exit if it is
+  check_git_branch
+
+  new_branch=$(create_git_branch_list $stream-stream-bump $base_branch)
+  switch_git_branch $base_branch $new_branch
+
+  update_WMCO_version $stream $wmco_version
+  update_OCP_version $stream $wmco_version
+
+  git submodule update --init --recursive
+  make bundle
```

Comment:
see comment on [test PR](https://github.com/openshift/windows-machine-config-operator/pull/1667#discussion_r1251081794), which is a consequence of make bundle